conf/config-default.yaml

```diff
@@ -450,4 +450,8 @@ plugin_attr:
       sequence_bits: 10               # each machine generates a maximum of (1 << sequence_bits) serial numbers per millisecond
       data_machine_ttl: 30            # live time for data_machine in etcd (unit: second)
       data_machine_interval: 10       # lease renewal interval in etcd (unit: second)
-
+  #proxy-mirror:
```

Comment:
What about uncommenting them and setting a default timeout (60000)?

Reply:
Agree, 60000 is the default value of nginx, uncommenting and setting them can be clearer. What about respects to nginx's 60 in seconds instead of 60000 in milliseconds ?